modules/livewrappedAnalyticsAdapter.js

```diff
@@ -62,6 +63,9 @@ let livewrappedAnalyticsAdapter = Object.assign(adapter({EMPTYURL, ANALYTICSTYPE
         if (!bidResponse.ttr) {
           bidResponse.ttr = time - bidResponse.start;
         }
+        if (!cache.bidAdUnits[bidResponse.adUnit]) {
```

Comment:
Yes, that is correct. What I do here is to keep track of all ad units that received at least one bid in the auction. The individual bids are handled elsewhere. Since the analytics adapter may send back bid responses to our end point in different requests, I needed a place to keep this particular information and make sure it is sent only once per adunit/auction since that make my life so much easier in the backend.

I guess I made a failed attempt to make it clearer by calling the metric "bidAdunits" and not "adUnitBids" :( Naming is the hardest problem in Computer Science...